client/src/pages/LibrarySearchPage/LibrarySearchPage.js

```diff
@@ -34,33 +40,25 @@ class LibrarySearchPage extends Component {
     this.setState({ deleteModal: false });
   };
 
-  closeDeleteSuccessModal = () => {
-    this.setState({ deleteSuccessModal: false });
-  };
-
-  deleteEntity = (type, id) => {
-    const { libraryActions } = this.props;
-    libraryActions.remove(id, this.deleteSuccess, this.deleteFailure);
-  };
-
   deleteFailure = () => {
     window.scrollTo(0, 0);
+    console.log('error');
     this.closeDeleteModal();
   };
 
   deleteSuccess = entity => {
     window.scrollTo(0, 0);
     this.closeDeleteModal();
     this.props.alertActions.hide();
-    this.setState(
-      {
-        deleteSuccessModal: entity.data,
-      },
-      function() {
-        //refresh data from the database - https://github.com/tannerlinsley/react-table/issues/808#issuecomment-373673915
-        this.table.fireFetchData();
-      },
+    this.props.alertActions.show(
+      'success',
+      'Success',
+      `${entity.data.name} was successfully deleted`,
     );
+    this.setState(function() {
```

Comment:
Is `this.setState` necessary since the state is no longer being changed?

Reply:
I'm not sure what that does. I didn't move it over to the component though. What do you recommend?

Reply:
@kmid5280 `setState` is a function used by React to update the state: https://css-tricks.com/understanding-react-setstate/. .Since you aren't passing any values for state changes, I would remove `this.setState` and only leave the call to `this.table.fireFetchData()`.

Reply:
Oh I know what setState does, I just wasn't sure what role it was playing in this particular case. If it's not needed I can go ahead and remove it.

Reply:
@kmid5280 Ah okay sorry I misunderstood there. Looking at the code that's being removed in the PR, setState was previously setting this in state:
```
     {
        deleteSuccessModal: entity.data,
      },
```
And since we've removed the success modal, setting the data for it is no longer necessary.